README.md

```diff
@@ -81,14 +81,14 @@ The full set of configurations is viewable on [the CI web site][jenkins].
 Compatibility fixes that do not cause newer versions to fail are welcome.
 
 [spack]: https://github.com/spack/spack
-[install]: doc/installation.rst
-[jenkins]: https://cloud.cees.ornl.gov/jenkins-ci/blue/organizations/jenkins/Celeritas/activity?branch=master
+[install]: doc/main/installation.rst
+[jenkins]: https://cloud.cees.ornl.gov/jenkins-ci/job/celeritas/job/develop
```

Comment:
Could you also add a link to the github actions? We've now got CI added for several additional compilers.